lib/streamlit/util.py

```diff
@@ -203,3 +216,37 @@ def extract_key_query_params(
             for item in sublist
         ]
     )
+
+
+K = TypeVar("K")
+V = TypeVar("V")
+
+
+class TimedCleanupCache(TTLCache, Generic[K, V]):
+    """A TTLCache that asynchronously expires its entries."""
+
+    def __init__(self, *args, **kwargs):
+        self._task: Optional[asyncio.Task[Any]] = None
```

Comment:
Is there a reason that it needs to be initialized at the first position? Wouldn't it be possible to do that after the `super().init` which I think is more common way of initializing class variables?